lib/core/ui/dashboard/home/home_date_element.dart

```diff
@@ -0,0 +1,52 @@
+import 'package:flutter/material.dart';
+import 'package:intl/intl.dart';
+import 'package:live_score/core/theme/app_theme.dart';
+
+class DashboardDateElement extends StatelessWidget {
+  const DashboardDateElement({
+    super.key,
+    required this.date,
+    this.isSelected = false,
+  });
+  final DateTime date;
+  final bool isSelected;
+
+  @override
+  Widget build(BuildContext context) {
+    DateFormat dayFormat = DateFormat('E');
```

Comment:
This logic is being re-run every time the widget is rebuilt, it is not needed. The better practise would be to move this logic to a function which would return `monthDay`. It should be placed outside the `build` method.

Reply:
Placed it in DateUtil file.